Add a bunch of new search operators (#674)
FreeFeed/freefeed-server: app/support/search/query-syntax.md

* in-body:
* in-comments:
* from:
* author: / by:
* in:
* in-my:
* commented-by:
* liked-by:

> * clicked-by:

Reviewer: ```suggestion
* cliked-by:
```

**to:user1,group2** limits search to posts published in group2 feed or written _to_ user1 as a direct message. This operator acts like **in:** for the groups but also allows to search in direct messages with the specific addressee.

> **clicked-by:user1,user2** limits search to comments licked by user1 or user2.

Reviewer: ```suggestion
**cliked-by:user1,user2** limits search to comments liked by user1 or user2.
```

> `cat clicked-by:alice` will find all comments liked by Alice with the "cat" word.

Reviewer: ```suggestion
`cat cliked-by:alice` will find all comments liked by Alice with the "cat" word.
```

> Since `clicked-by:` makes sense only for comments, it switches the search scope to comments. So the query `cat clicked-by:alice` is equal to `in-comments: cat clicked-by:alice`. Being used in post body scope (like `in-body: clicked-by:...`), `clicked-by:` is ignored.

Reviewer: ```suggestion
Since `cliked-by:` makes sense only for comments, it switches the search scope to comments. So the query `cat cliked-by:alice` is equal to `in-comments: cat cliked-by:alice`. Being used in post body scope (like `in-body: cliked-by:...`), `cliked-by:` is ignored.
```
